Add contact form mobile version.

Drod86 opened the pull request:
<h2>In this branch we:</h2>

- Added HTML markup and CSS styles for:
  - footer
  - form heading
  - form element
  - social media icons
  - home-indicator

stanleySimeon commented on README.md:
As mentioned above, kindly remove the live demo section if your project does not deploy yet.

- Don't forget that, never commit commented code in your code base to always keep professionalism.

stanleySimeon commented on index.html:
- Kindly make sure the hamburger icon is inside a `nav` tag.
- Please make sure your stick to the design as much as you can to follow the Figma design.

> ## Your design
>
><img width="301" alt="Screenshot 2023-06-02 at 8 05 05 PM" src="https://github.com/Drod86/My_Portfolio/assets/91237525/b592126d-ba9a-413d-8c21-5e7f973300b8">
>
> ## The Figma design
>
><img width="368" alt="Screenshot 2023-06-02 at 8 06 08 PM" src="https://github.com/Drod86/My_Portfolio/assets/91237525/7ad50ab1-b7f5-4111-8e28-24b0f562b4b9">

stanleySimeon commented on index.html:
- Please, I suggest you use a `ul` and `li` tag instead of a `nav` tag to list the social icons for good semantics reasons and best practices. 😄 

- As I mentioned above, kindly make sure you stick to the design as much as possible.
I suggest you fix the height of the home section to make it match the Figma design.

> ## Your Design
>
><img width="300" alt="Screenshot 2023-06-02 at 8 11 29 PM" src="https://github.com/Drod86/My_Portfolio/assets/91237525/cc069b76-2c22-4697-baf6-ab5cef3f109e">
>
> ## The Figma Design
>
><img width="369" alt="Screenshot 2023-06-02 at 8 12 41 PM" src="https://github.com/Drod86/My_Portfolio/assets/91237525/7ffe31a3-b715-45db-ba76-9736695baa88">

stanleySimeon reviewed (changes requested):
# Hi @Drod86, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://media3.giphy.com/media/3orifcRJpsEy8fJbZ6/giphy-downsized.gif?cid=29caca75gpeopdt7l9sxgwk3ws2oooiguj6g9o6pnod67wfe&ep=v1_gifs_search&rid=giphy-downsized.gif&ct=g)

## Highlights
- Correct use of Gitflow ✔️
- Linter checks are passed ✔️
- Good PR description ✔️
- Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- adding your live demo link

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Drod86 commented:
Hello @stanleySimeon,
Thank you for your feedback. I made the following changes:

- Removed all commented code
- Removed unused section in README
- Adjusted the size of the UI elements to more accurately mirror the Figma template
- Placed the hamburger menu image in a nav element
- Placed social media icons in an unordered list element

franclobo reviewed (approved):
Hi @Drod86, 

# STATUS: APPROVED 😃 

![git-merge](https://github.com/Drod86/My_Portfolio/assets/58642949/54443d9f-e861-419b-bd34-d58bc5abecaf)

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Good UI ✔️ 
- Best practices of HTML & CSS ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Rafin86 committed:
Add desktop size media quieries

Rafin86 committed:
Add contact form styles

Rafin86 committed:
Replace hard coded color values with css variables

Rafin86 committed:
Add media quiery brakepoints

Rafin86 committed:
Remove unused code and fix linter errors

WilfriedM01 committed:
Add html form architecture

Rafin86 committed:
Remove list styles

Rafin86 committed:
Add form styles. Add footer socials and home-indicator sections

Rafin86 committed:
Fix stylelint error

tresorsawasawa reviewed (changes requested):
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Drod86 and @WilfriedM01 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Rafin86 committed:
Change form textarea font-family to match the design.

AbelG101 reviewed (approved):
![](https://media.giphy.com/media/eKfovWZOlOpbTDEX9F/giphy.gif)

# Hi @Drod86 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- The UI is looking top-notch! ✔
- Form inputs are properly validated  ✔
- Correct use of GitHub Flow ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Optional suggestions
- Check the comments below...

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
